geps/gep-1867/index.md

```diff
@@ -75,43 +75,74 @@ For example, if I wanted to represent a `version` field and change that to trigg
 
 ## API
 
-In order to address the concerns above, I propose a standard `infrastructure` API is added to `Gateway` and `GatewayClass`.
-Note the important part of this is the `Gateway` change; the `GatewayClass` aspect is mostly for consistency.
+In order to address the concerns above, I propose a standard `infrastructure` API is added to `Gateway`.
 
 The exact fields are out of scope for this GEP and will be handled by additional GEPs.
 One example GEP already depending on this is [GEP-1651](/geps/gep-1651).
 
-The fields as defined below are, of course, not useful.
-This is intended as a basis for other PRs, not to provide value on its own.
-This GEP will remain in provisional until at least one field is ready to be promoted.
+The fields as defined below are, at time of writing, the existing fields covered by this GEP. More fields may be added as needed.
 
 ```go
 type GatewaySpec struct {
   // Infrastructure defines infrastructure level attributes about this Gateway instance.
+	//
+	// Support: Extended
+	//
+	// +optional
   Infrastructure GatewayInfrastructure `json:"infrastructure"`
   // ...
 }
 
-type GatewayClassSpec struct {
-  // Infrastructure defines infrastructure level attributes for all Gateways in this class.
-  // A Gateway may provide configuration for the same values; as all fields in GatewayInfrastructure are implementation specific,
-  // the merging logic between these is as well. However, the GatewayClass is generally expected to be providing defaults
-  // rather than overrides.
-  Infrastructure GatewayClassInfrastructure `json:"infrastructure"`
-  // ...
-}
-
 type GatewayInfrastructure struct {
-  // ParametersRef provides a arbitrary implementation-specific configuration for
-  // fields not expressed directly in this struct.
-  // This follows the same semantics as GatewayClass's ParametersRef, but lives on the Gateway.
-  ParametersRef ParametersReference
+  // Labels that SHOULD be applied to any resources created in response to this Gateway.
+	//
```

Comment:
Tiny nit: Looks like a combination of tabs and spaces happening here and elsewhere in md file now.